index.html

```diff
@@ -47,36 +64,43 @@ <h4>LET'S CONNECT</h4>
                 </a>
             </li>
             <li>
-                <a title="The link will take you to my instagram profile" href="https://www.instagram.com/mpdaddympd/">
+                <a title="The link will take you to my instagram profile"
+                    href="https://www.instagram.com/mpdaddympd/">
                     <i class="fa fa-instagram social"></i>
                 </a>
             </li>
             <li>
-                <a title="The link will take you to my linkin profile" href="https://www.linkedin.com/in/anold-mwawasi">
+                <a title="The link will take you to my linkin profile"
+                    href="https://www.linkedin.com/in/anold-mwawasi">
                     <i class="fa fa-linkedin social"></i>
                 </a>
             </li>
         </ul>
+    </article>
     </section>
+
     <section class="work_section">
         <article class="sections">
-            <div class="scrnshot">
-                <img src="images/Snapshoot Portfolio (2).png" alt="">
+            <div class="scrnshot width">
+                <img class="mobile" src="images/Snapshoot Portfolio (2).png" alt="">
+                <img class="desktop" src="images/Snapshoot Portfolio - Copy.png" alt="">
             </div>
-            <h2 class="colorh2">Tonic</h2>
-            <div>
+            <div class="width">
+                <h2 class="colorh2">Tonic</h2>
                 <div class="canopy">
                     <h4 class="colorh4">CANOPY</h4>
                     <ul>
-                        <li>
+                        <li class="canopylist">
                             <span>
                                 <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
```

Comment:
- [ ] kindly consider giving your icons some border-radius and the right background color  to make them look exactly like the one provided in the design you may also download them from Figma to make it quick  and doing so will demonstrate your skills in implementing the provided design exactly check the images below to easily see the difference 
- [ ] kindly consider also giving the `see project` button the same color as the one given in Figma to make it an exact duplicate of the given one 
- your design 👇 
<img width="735" alt="Screen Shot 2023-05-12 at 15 25 10" src="https://github.com/MPDADDY/portfolio/assets/60146030/d5b8ce3b-bda0-409d-9abc-01cda8dbbc7d">


- Figma design 👇 
<img width="802" alt="Screen Shot 2023-05-12 at 15 25 38" src="https://github.com/MPDADDY/portfolio/assets/60146030/80bebb8d-6556-46be-81a8-5f51b7488440">